### ISSUE TITLE
bug: Getting error in Android Studio

### ISSUE THREAD

**Issue by mayankkataria** [ionic-team/capacitor#4795]
> Caused by: java.lang.NoSuchMethodError: org.gradle.api.artifacts.result.ComponentSelectionReason.getDescription()Ljava/lang/String;

I'm facing this error in Android Studio. My app is running perfectly through live server when I run this command ```ionic cap run android -lc --external``` but I can't run that through android studio. And the reason I want it to run in AS is for deployment purpose. If there's any alternative way to deploy through terminal than please help.
Thanks.

**Comment by Ionitron** [comment]
This issue may need more information before it can be addressed. In particular, it will need a reliable Code Reproduction that demonstrates the issue.

Please see the Contributing Guide for [how to create a Code Reproduction](https://github.com/ionic-team/capacitor/blob/HEAD/CONTRIBUTING.md#creating-a-code-reproduction).

Thanks!
Ionitron 💙

**Comment by mayankkataria** [comment]
I solved this issue by updating Android Studio. Actually this error was saying "Failed to notify build listener" which can be solved by updating Android Studio [(source)](https://stackoverflow.com/questions/56504853/how-to-fix-failed-to-notify-build-listener-error). I followed the updating steps from [here](https://stackoverflow.com/questions/28133518/how-to-update-android-studio-automatically). The older version of AS can't run new ionic projects, kindly update to latest version. Hope it would help someone.
Thanks.
